commcare/src/main/resources/webapp/partials/settings.html

```diff
@@ -23,6 +23,9 @@
              <a ng-if="selectedConfig" ng-disabled="!validateConfig()" ng-click="validateConfig() && verify()" type="button" class="btn btn-default">
                  <i class="fa fw fa-check-circle-o"></i> {{msg('commcare.verify')}}
              </a>
+             <a ng-if="selectedConfig" ng-disabled="!syncAllowed()" ng-click="syncConfig()" type="button" class="btn btn-default">
```

Comment:
We should add "syncAllowed" condition to ng-click, since I can currently click the button, even though it appears disabled.